src/principal/service.ts

```diff
@@ -411,8 +411,8 @@ function recordToModel(user: PrincipalsRecord): Principal {
     href: `/${userTypeIntToUserType(user.type)}/${user.external_id}`,
     externalId: user.external_id,
     nickname: user.nickname!,
-    createdAt: new Date(user.created_at),
-    modifiedAt: new Date(user.modified_at),
+    createdAt: new Date(+user.created_at),
+    modifiedAt: new Date(+user.modified_at),
```

Comment:
Both fields are stored as BigInt in postgres. But when `knex` (or upstream `pg`) queries that data, a string version of the number (e.g., `"179289..."`) is returned. 

This may not be an issue for mysql and sqlite. If that's the case the `+` is a no-op.

Reply:
Thanks for finding this. This is going to be annoying going forward with postgres =(